solutions/secure-cross-regional-bucket/variables.tf

```diff
@@ -162,3 +162,21 @@ variable "retention_permanent" {
   type        = bool
   default     = false
 }
+
+variable "object_locking_enabled" {
+  description = "Specifies if an object lock configuration should be created. Requires 'object_versoning_enabled' to be true. Only used if 'create_cos_bucket' is true."
+  type        = bool
+  default     = false
+}
+
+variable "object_lock_duration_days" {
+  description = "Specifies the default number of days for the retention lock duration. Only used if 'create_cos_bucket' is true."
```

Comment:
maybe add to the description only set days or years

Reply:
Added